vividus-extension-selenium/build.gradle

```diff
@@ -21,6 +21,9 @@ project.description = 'Vividus extension for selenium'
 
      api(group: 'ru.yandex.qatools.ashot', name: 'ashot', version: versions.ashot)
 
+     implementation(group: 'com.google.zxing', name: 'core', version: zxingVersion)
+     implementation(group: 'com.google.zxing', name: 'javase', version: zxingVersion)
+
```

Comment:
```suggestion
     implementation platform(group: 'com.google.zxing', name: 'zxing-parent', version: '3.4.1')
     implementation(group: 'com.google.zxing', name: 'core')
     implementation(group: 'com.google.zxing', name: 'javase')
```

?

Reply:
also please group dependencies for different configuration: all `api`, then all `implementation`